src/openrct2/ride/Ride.cpp

```diff
@@ -4588,16 +4577,14 @@ static void ride_create_vehicles_find_first_block(Ride* ride, CoordsXYE* outXYEl
 
     assert(curTrackElement != nullptr);
 
-    int32_t x = curTrackPos.x;
-    int32_t y = curTrackPos.y;
+    CoordsXY trackPos = curTrackPos;
```

Comment:
variable not required.

Reply:
Inside the loop we compare `trackPos` (which is often updated) with `curTrackPos` which is the initial one. Should we really remove it?